.gitlab-ci.yml

```diff
@@ -1,29 +1,30 @@
-image: openjdk:11
-
-workflow:
-  rules:
-    - if: $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == 'dev'
+include:
+  remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
 
 stages:
   - build
-  - test
+  - unit-test
+  - release-dev
+  - release-master
+  - deploy-dev
+  - integration-dev
+  - deploy-staging
+  - integration-staging
+  - deploy-prod
+  - integration-prod
 
 cache:
   paths:
     - .gradle/wrapper
     - .gradle/caches
 
-
-build:
-  stage: build
-  script:
-    - ./gradlew
-    
-test:
-  stage: test
+Unit Test:
```

Comment:
Why is this one Title Case while all the others kebab-case?

Reply:
Snake case if for the stage name and title case is for the job name. The job name is what shows in the pipeline UI so title case looks nicer. You can see https://github.com/ebi-ait/gitlab-ci-templates/blob/master/build-release-deploy.yml for full details of the jobs